[SITE-2496] D7ES added to drops-7 (#9346)
pantheon-systems/documentation: source/releasenotes/2024-12-10-drupal-7-d7es.md

---

> title: Tag1 Extended Support module installed on Drupal 7 upstream

Reviewer: In an internal draft we had the title here as 

```
Drupal 7 Long-Term Support now operational on Pantheon
```

I think it is wise to immediately say in the prose of the release note that the specific observable change is the inclusion of the module. But to my knowledge there are a few more internal things we've done (that don't necessarily need to be spelled out in the release note) that add up "the whole idea is operational"

Reviewer 2: ```suggestion
title: Drupal 7 Long-Term Support now operational on Pantheon
```

published_date: "2024-12-05"
categories: [drupal, action-required]
---

> The default Pantheon Drupal 7 upstream has been updated to include the Tag1 Drupal 7 Extended Support (D7ES) module.

Reviewer 3: … "required to take advantage of Pantheon's Long Term Support (LTS) for Drupal 7."

> The default Pantheon Drupal 7 upstream has been updated to include the Tag1 Drupal 7 Extended Support (D7ES) module.

Reviewer: ```suggestion
The default Pantheon Drupal 7 upstream has been updated to include [the Tag1 Drupal 7 Extended Support (D7ES) module](https://www.drupal.org/project/tag1_d7es). This module is required to take advantage of [Pantheon's Long Term Support (LTS) for Drupal 7](https://pantheon.io/blog/pantheon-and-tag1-support-drupal-7-websites).
```

Reviewer 2: ```suggestion
The default [Pantheon Drupal 7 upstream](https://github.com/pantheon-systems/drops-7) has been updated to include [the Tag1 Drupal 7 Extended Support (D7ES) module](https://www.drupal.org/project/tag1_d7es). This module is required to take advantage of [Pantheon's Long Term Support (LTS) for Drupal 7](https://pantheon.io/blog/pantheon-and-tag1-support-drupal-7-websites).
```

> Future LTS updates to both Drupal 7 Core and the D7ES module will be available as drops-7 upstream updates. Learn more here: https://docs.pantheon.io/supported-drupal#d7-core-lts-and-platform-compatibility 

Author: ```suggestion
Future LTS updates to both Drupal 7 Core and the D7ES module will be available as drops-7 upstream updates. Learn more in our [Drupal 7 LTS documentation](https://docs.pantheon.io/supported-drupal#d7-core-lts-and-platform-compatibility).
```

> Future LTS updates to both Drupal 7 Core and the D7ES module will be available as drops-7 upstream updates. Learn more in our [Drupal 7 LTS documentation](https://docs.pantheon.io/supported-drupal#d7-core-lts-and-platform-compatibility).

Reviewer 3: Should we be consistent in calling the upstream either the "Pantheon Drupal 7 Upstream" or "drops-7" vs switching?

## Action Required

> To receive security ongoing security coverage for your contrib modules, apply this upstream update then enable the Tag1 D7ES module.

Reviewer 2: Non-blocking but it would be nice if there were some screenshots of the module in the modules list either here in the release note or in the docs about the D7ES module.

Reviewer: Should we refer to it as "the Tag1 D7ES module" or something like "the Drupal 7 Extended Support module?" I think we should go with whatever text appears in the module listing in the Drupal interfaces. 

So yeah, to @jazzsequence's point, this would be a good spot to have a screenshot of the module installation interface with the checkbox toggled to on for this module.

Reviewer 4: FWIW it's Tag1 D7ES https://docs.tag1.com/setup/#how-to-test

Reviewer: And after someone has done this step, is there something they will see or can do that verifies for them that the whole thing is working?

Reviewer 4: If they want to fully test, they can follow the docs link from Tag1. We can include that in a docs update.

Reviewer 2: I can spin this up and get the relevant screenshots. I'm not sure there's anything to show until there's an update to a module but the Tag1 docs do show what that looks like in practice.

Reviewer 2: Noting that we don't link out to the Tag1 docs site for D7ES in our documentation. We should, if we don't want to rewrite 90% of it from a Pantheon context.

> 

Reviewer: In our [internal notes](https://docs.google.com/document/d/17Fs08vSLoRShdvPFGzRz_KCuY9rFke8QA3-3KTMNn4s/edit?tab=t.0) we also called out the need/suggestion of configuring email alerts

Reviewer 2: That's referenced in the Tag1 documentation (which we link to). Our docs do not talk about setting up email alerts. We should add that as a heading to the docs page and just link to https://docs.tag1.com/notifications/

Reviewer 5: We should replace the Tag1 documentation with our own. The Tag1 version instructs the user to do things not necessary on Pantheon. What's more, it asks the user to look up information in the module that is not displayed when on Pantheon, which could cause confusion for users.

Reviewer 2: Our docs do talk about configuring email alerts. I added a screenshot and a paragraph about notifications.

## Long-Term Support

Pantheon is committed to providing security updates for Drupal 7 for at least two additional years beyond its upcoming end-of-life date on January 5, 2025. This commitment is in partnership with Tag1 Consulting, offering Long-Term Support (LTS) for Drupal 7 at no additional cost.

> 

Reviewer: @scottbuscemi @pwtyler somewhere in here can you add a sentence or two about how our LTS intersects with Autopilot?

Reviewer 2: According to our [documentation](https://docs.pantheon.io/supported-drupal#d7es-usage) "[Autopilot](https://docs.pantheon.io/guides/autopilot) will automatically apply contrib module updates from the Tag1 D7ES service the next time Autopilot runs."

Since it's in our docs, I don't think it's necessary to add to the release note.

> For details about this offering and steps to prepare your sites, refer to our [related documentation](/supported-drupal/#drupal-7-long-term-support).

Reviewer 4: ```suggestion
For details about this offering and how to configure your sites, refer to our [related documentation](/supported-drupal/#drupal-7-long-term-support).
```

---
title: Drupal 7 Long-Term Support now available on Pantheon
published_date: "2024-12-05"
categories: [drupal, action-required]
---

> [Pantheon's Long Term Support (LTS) for Drupal 7](https://pantheon.io/blog/pantheon-and-tag1-support-drupal-7-websites) is now available. The default [Pantheon Drupal 7 upstream](https://github.com/pantheon-systems/drops-7) has been updated to include [the Tag1 Drupal 7 Extended Support (D7ES) module](https://www.drupal.org/project/tag1_d7es). This module is essential for leveraging our ongoing LTS offering.

Reviewer 2: ```suggestion
[Pantheon's Long-Term Support (LTS) for Drupal 7](https://pantheon.io/blog/pantheon-and-tag1-support-drupal-7-websites) is now available. The default [Pantheon Drupal 7 upstream](https://github.com/pantheon-systems/drops-7) has been updated to include [the Tag1 Drupal 7 Extended Support (D7ES) module](https://www.drupal.org/project/tag1_d7es). This module is essential for leveraging our ongoing LTS offering.
```

### What's new

With this update, future LTS releases for Drupal 7 Core and the D7ES module will be provided through Pantheon’s Drupal 7 upstream. For details, see our [Drupal 7 LTS documentation](/supported-drupal#d7-core-lts-and-platform-compatibility).

### Action required

To continue receiving security coverage for contributed modules:

* Apply the latest upstream update to your Drupal 7 site.

> * Enable the Tag1 D7ES module in your environment.

Reviewer 5: We should also add a bullet here instructing the user to configure a notification email address from the Tag1 D7ES configuration page. The config page is at `/admin/config/system/tag1-d7es`

Reviewer 4: It'd be better for us to point folks to docs rather than include all the steps in the release note. Otherwise we'll have to maintain this release note if anything changes.

Reviewer 2: Agree that change should go in the docs. This PR includes both, however, so it can be updated at the same time.
